stdpopsim/slim_engine.py

```diff
@@ -932,14 +959,14 @@ def fix_time(event):
     T = [round(e.start_time * demographic_model.generation_time) for e in epochs]
     migration_matrices = [e.migration_matrix for e in epochs]
 
-    N = np.empty(shape=(dd.num_populations, len(epochs)), dtype=int)
+    N = np.empty(shape=(dd.num_populations, len(epochs)), dtype=float)
     growth_rates = np.empty(shape=(dd.num_populations, len(epochs)), dtype=float)
     for j, epoch in enumerate(epochs):
         for i, pop in enumerate(epoch.populations):
             # SLiM simulates a diploid population, so rescale population size
             # depending on contig ploidy. If/when the SLiM simulation is
             # converted to a haploid model, this rescaling should be removed.
-            N[i, j] = int(pop.end_size * contig.ploidy / 2)
+            N[i, j] = pop.end_size * contig.ploidy / 2
```

Comment:
This should be getting covered by the tests, but codecov disagrees ... ?

Reply:
Nevermind-- codecov itself shows these as covered, the github API isn't picking up on that for some reason.